src/org/jgroups/stack/GossipRouter.java

```diff
@@ -967,14 +990,15 @@ public static void main(String[] args) throws Exception {
                 default:
                     break;
             }
+            socketFactory.setServerSocketConfigurator(s -> ((SSLServerSocket)s).setSSLParameters(serverParameters));
             router.socketFactory(socketFactory);
+            type = String.format(" (%s/%s)", tls_protocols!=null?Arrays.toString(tls_protocols):"TLS", context.getProvider().getName());
```

Comment:
nitpick, use `String.join()` instead of `Arrays.toString()`
```suggestion
            type = String.format(" (%s/%s)", tls_protocols!=null?String.join(",", tls_protocols):"TLS", context.getProvider().getName());
```

Reply:
Done